test/REGRESSIONS

```diff
@@ -426,6 +410,10 @@ Reviewed 2015-01-23
 
 === sporadic failures below ===
 
+PGC-W-0205-Argument mismatch for assert - all but a few failed
+--------------------------------------------------------------
+(xe-wb.prgenv-pgi 2015-01-29)
```

Comment:
I was following the style of the entry below it, since the initial failure seemed similar to that sporadic case, but you're right, the error does seem more specific to pgi